mesoscope/cellpack
# Feature/upload to s3

Problem
=======
What is the problem this work solves, including
closing #140 

Solution
========
What I/we did to solve this problem
- In `simularium_helper.py`, added functionalities to store results to S3 - `cellpack-results/simularium` , upload metadata to firebase, and open packing results in simularium app.   

with @meganrm @mogres 

## Type of change
Please delete options that are not relevant.
* New feature (non-breaking change which adds functionality)


Steps to Verify:
----------------
1. pack a local recipe: `pack -r [path-to-a-local-recipe] -c [path-to-a-config-file]`
2. pack a remote recipe: pack -r `firebase:recipes/[firebase-recipe-id] -c [path-to-a-config-file]`(make sure you have the recipe living in your firebase before packing)


Keyfiles (delete if not relevant):
-----------------------
1.`simularium_helper.py`
2.`AWSHandler.py`
